cmd/tk/flags.go

```diff
@@ -13,14 +13,16 @@ import (
 )
 
 type workflowFlagVars struct {
-	name    string
-	targets []string
+	name                  string
+	targets               []string
+	jsonnetImplementation string
 }
 
 func workflowFlags(fs *pflag.FlagSet) *workflowFlagVars {
 	v := workflowFlagVars{}
 	fs.StringVar(&v.name, "name", "", "string that only a single inline environment contains in its name")
 	fs.StringSliceVarP(&v.targets, "target", "t", nil, "Regex filter on '<kind>/<name>'. See https://tanka.dev/output-filtering")
+	fs.StringVar(&v.jsonnetImplementation, "jsonnet-implementation", "go", "Use `go` to use native go-jsonnet implementation and `binary:<path>` to delegate evaluation to a binary (with the same API as the regular `jsonnet` binary)")
```

Comment:
Is `binary` merged?

Reply:
Oh, no, but it'll definitely be before a release so 🤷